EIPS/eip-1.md

```diff
@@ -168,37 +246,45 @@ if neither the email address nor the GitHub username are given.
 
 At least one author must use a GitHub username, in order to get notified on change requests and have the capability to approve or reject them.
 
-### `discussions-to` header
+#### `discussions-to` header
 
 While an EIP is a draft, a `discussions-to` header will indicate the URL where the EIP is being discussed.
 
 The preferred discussion URL is a topic on [Ethereum Magicians](https://ethereum-magicians.org/). The URL cannot point to Github pull requests, any URL which is ephemeral, and any URL which can get locked over time (i.e. Reddit topics).
 
-### `type` header
+#### `type` header
 
 The `type` header specifies the type of EIP: Standards Track, Meta, or Informational. If the track is Standards please include the subcategory (core, networking, interface, or ERC).
 
-### `category` header
+#### `category` header
 
 The `category` header specifies the EIP's category. This is required for standards-track EIPs only.
 
-### `created` header
+#### `created` header
 
 The `created` header records the date that the EIP was assigned a number. Both headers should be in yyyy-mm-dd format, e.g. 2001-08-14.
 
-### `requires` header
+#### `requires` header
 
 EIPs may have a `requires` header, indicating the EIP numbers that this EIP depends on. If such a dependency exists, this field is required.
 
 A `requires` dependency is created when the current EIP cannot be understood or implemented without a concept or technical element from another EIP. Merely mentioning another EIP does not necessarily create such a dependency.
 
-## Linking to External Resources
+### Linking to External Resources
 
 Other than the specific exceptions listed below, links to external resources **SHOULD NOT** be included. External resources may disappear, move, or change unexpectedly.
 
 The process governing permitted external resources is described in [EIP-5757](./eip-5757.md).
 
-### Execution Client Specifications
+External resources not so permitted **MAY** be included *at the Editors' discretion*.
+
+A complete reference to every external resource **MUST** be included in a Normative or Informative References section.
+
+We take several precautions to ensure the continued accessibility of all external resources.  First, we follow the requirements of the [RFC 7322 RFC Style Guide](https://www.rfc-editor.org/rfc/rfc7322.html): references **MUST** be full citations, including author(s), title, and publication information -- URLs are acceptable, but **MUST NOT** be the sole information provided for a reference.  In addition, Links **SHOULD** meet the Requirements for Origins in [EIP-5757](./eip-5757.md), and a Digital Object Identifier (DOI), where available, **MUST** be included. Within "fair use", short quotations from referenced works are encouraged, as embedded quotations will not change. 
```

Comment:
I'd much prefer we use a computer tractable citation format.

There is an example of how `csl-json` looks when rendered in [EIP-6963](https://eips.ethereum.org/EIPS/eip-6963#fn:rfc4122), and [the relevant source](https://github.com/ethereum/EIPs/blob/e4cb6fda2825de5e8afbe27d33789f9558faaf8d/EIPS/eip-6963.md?plain=1#L248-L280).

You can convert between several well known formats (like BibTeX) here: https://citation.js.org/demo/ (choose CSL-JSON as output format.)

> URLs are acceptable, but **MUST NOT** be the sole information provided for a reference

Why not? If the URL contains a git commit hash or points to an exact document (like links to RFCs), I don't see why the URL alone is insufficient. We're an internet-first standards body. If anyone prints EIPs, I'd be quite surprised.

Reply:
I'd also prefer a computer-tractable citation format, but didn't try to specify one here.  For the URLs which we have "blessed" we can probably generate citations automatically.
 
 I took need for a non-URL reference from the IETF, which is the original internet-first standards body, and they apply the rule to their own standards.  As ever, the Editors can make exceptions if there is no other choice, but most documents at least have authors and titles.   Given that (and any other publication info) it is much easier to find a copy if the link goes dead.  And I don't see requiring an URL where a document is published but doesn't have a stable URL.  But I'm not a stickler on details here -- just the policy of "as complete a citation as possible."

Reply:
I feel pretty strongly that someone with only an internet connection and a web browser should be able to view an EIP and all documents referenced therein.

Reply:
For reference, CSL did create some confusion so I moved the linking to just the RFC link here. I like the way the CSL is getting rendered, but the way the IETF handles normative referencing seems to also work. If there's a way to easily generate a CSL-JSON object from a URL that would be ideal. I couldn't figure this out with the demo link provided above @SamWilsn.

Reply:
https://citation.js.org/demo/ is how I format it. Doesn't automatically support pulling from a URL, but you can get, for example, bibtex and convert it to CSL-JSON.

Reply:
> I feel pretty strongly that someone with only an internet connection and a web browser should be able to view an EIP and all documents referenced therein.

I agree, they should, especially for normative links.  They should have enough information  to find the document in a library, and for a successful internet search if the link goes dead.